symbols/arm9.yml

```diff
@@ -38,6 +50,26 @@ arm9:
         The entrypoint for the ARM9 CPU. This is like the "main" function for the ARM9 subsystem.
         
         No params.
+    - name: MIiUncompressBackward
+      address:
+        NA: 0x2000970
+      description: Startup routine in crt0.
```

Comment:
What is crt0? Can we add more details in the description?

Reply:
It's a standard set of startup routines for C programs. https://en.wikipedia.org/wiki/Crt0

I can link this in the descriptions.